hie-base/Haskell/Ide/Engine/PluginTypes.hs

```diff
@@ -18,8 +18,14 @@ module Haskell.Ide.Engine.PluginTypes
   , ParamValP(..)
   , ParamMap
   , pattern ParamTextP
+  , pattern ParamIntP
+  , pattern ParamBoolP
   , pattern ParamFileP
```

Comment:
ParamFileP and ParmaPosP are using the LSP types under the hood, and are used for passing parameters to the requests.